src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+
+.calendar {
+  width: $widthCalendar;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    width: $widthDay;
+    height: $widthDay;
+    border: 1px solid black;
+    background-color: $colorDay;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
```

Comment:
Will be better to set this property using `*` selector as it will be applied for all elements in your project